python/sports-league-scheduling/src/sports_league_scheduling/demo_data.py

```diff
@@ -0,0 +1,126 @@
+import json
+import random
```

Comment:
To prevent accidentally using a shared random, only import the `Random` class from the package.

```suggestion
from random import Random
```